tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -226,10 +226,11 @@ inline auto ArbitraryBaseAvifDecoder() {
 // options fuzzed, with the exception of 'ignoreColorAndAlpha' (because it would
 // break most tests' assumptions).
 inline auto ArbitraryAvifDecoderWithGainMapOptions() {
-  return fuzztest::Map(
-      AddGainMapOptionsToDecoder, ArbitraryBaseAvifDecoder(),
-      /*enable_parsing_gain_map_metadata=*/fuzztest::Arbitrary<bool>(),
-      /*enable_decoding_gain_map=*/fuzztest::Arbitrary<bool>());
+  return fuzztest::Map(AddGainMapOptionsToDecoder, ArbitraryBaseAvifDecoder(),
+                       /*gain_map_decode_mode=*/
+                       fuzztest::ElementOf({GainMapDecodeMode::kDontDecode,
```

Comment:
Nit/optional: `fuzztest::Values` should be enough here

Reply:
As discussed offline, there's no such function in fuzztest :)